src/test/kotlin/uk/gov/justice/digital/hmpps/hmppselectronicmonitoringdataplatformapi/controller/DeviceWearerControllerTest.kt

```diff
@@ -41,12 +41,11 @@ class DeviceWearerControllerTest {
 
   @Test
   fun `getAllDeviceWearers should return Ok with a message when there are no device wearers`() {
-    Mockito.`when`(deviceWearerService.getAllDeviceWearers()).thenReturn(listOf<DeviceWearer>())
```

Comment:
@PedroSoaresMoj I think this was a you change - can you comment on the reason? Seems like it makes this test do nothing, when combined with the change on old-line 49.